doc/06-distributed-monitoring.md

```diff
@@ -3227,6 +3227,53 @@ information/pki: Writing certificate to file 'icinga2-satellite1.localdomain.crt
 
 Copy and move these certificates to the respective instances e.g. with SSH/SCP.
 
+#### External CA/PKI
+
+Icinga works best with its own certificates.
+The commands described above take care of the optimal certificate properties.
+Also, Icinga renews them periodically at runtime to avoid expiry.
+But you can also provide your own certificates,
+just like to any other application which uses TLS.
+
+!!! warning
+
+    The only serious reasons to generate own certificates are company policies.
+    You are responsible for making Icinga working with your certificates,
+    as well as for [expiry monitoring](10-icinga-template-library.md#plugin-check-command-ssl_cert)
+    and renewal.
+    
+    Especially `icinga2 pki` CLI commands don't expect such certificates.
+    
+    Apropos renewal, do not provide the CA private key anywhere!
+    Icinga uses it to automatically renew certificates
+    with standard properties, not any custom ones.
```

Comment:
This paragraph is a bit hard to understand as it does not directly communicates the concrete issue. How about:

> Do not provide your custom CA private key to Icinga 2, or it will attempt to renew certificates with the default properties.

Reply:
I finetuned your suggestion a bit.